tf/taskcluster/audit-logs.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_cloudwatch_log_group" "taskcluster_audit_logs" {
 }
 
 resource "aws_cloudwatch_log_stream" "taskcluster_audit_logs" {
-  name           = "S3Delivery"
+  name           = "${var.dpl}-s3-delivery"
```

Comment:
These are actually nested in a LogGroup, so the lack of a unique name wasn't a problem.  However, they can't be `terraform import`ed either.  This change could *probably* be dropped, but maybe we just keep it in anyway?

Reply:
I guess instead of importing this, I could have just deleted the resource via the S3 console and re-run terraform apply. OK, OK, fine I'll remove this hunk.